routes/index.js

```diff
@@ -1,4 +1,8 @@
 const express = require("express");
 const router = express.Router();
 
-module.exports = router;
\ No newline at end of file
+const signUpRouter = require("./signUpRouter");
+
+router.use("/signUp", signUpRouter.router);
```

Comment:
```suggestion
router.use("/signup", signUpRouter.router);
```
보통 URI만들 때 소문자와 dash("-")를 이용해서 많이 만듭니다. 
그런데 signup의 경우 통상적으로 붙여서 만듭니다.

Reply:
전체적으로 수정해보았습니다.